src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/coding/innerassignment/InputInnerAssignmentWithEnhancedSwitch.java

```diff
@@ -0,0 +1,22 @@
+/*
+InnerAssignment
+
+
+*/
+
+//non-compiled with javac: Compilable with Java14
+package com.puppycrawl.tools.checkstyle.checks.coding.innerassignment;
+
+public class InputInnerAssignmentWithEnhancedSwitch {
+
+    void method(final String operation) {
+        boolean flag = false;
+        switch (operation) {
+            case "Y" -> flag = true;  // ok
```

Comment:
Please remove trailing comment `// ok`
I know there are a lot of examples where we have them, but it is our mistake of past.
No `// violation` means ok, so `//ok` is just a noise.